### ISSUE TITLE
Multi-line list items items triggers MD029

### ISSUE THREAD

**Issue by teodorlu** [DavidAnson/markdownlint#95]
Hello,

Markdown allows breaking lines in list items. This improves readability when reading the source in an editor that doesn't break lines. Example:

```markdown
1. **The first item** may be described by a very long line, that naturally wraps
   when it passes some character limit.
2. **The second line** thus continues, and common markdown renderers seem to
   handle this nicely.
```

renders quite nicely with Github:

1. **The first item** may be described by a very long line, that naturally wraps
   when it passes some character limit.
2. **The second line** thus continues, and common markdown renderers seem to
   handle this nicely.

... but triggers MD029 in the linter. MD029 seems to be for something else.

**Comment by teodorlu** [comment]
Sidenote:

- Github seems to add a `<br>` between the lines.
- Your [live demo](https://dlaa.me/markdownlint/) does not.
- Pandoc does not.

**Comment by DavidAnson** [comment]
MD029 is correctly reporting here, please see: https://github.com/DavidAnson/markdownlint/issues/45

However, because this comes up occasionally, I have updated the message to try to clarify that the default numbering behavior is 1/1/1. This will be part of the next release.

**Comment by teodorlu** [comment]
Sorry, my bad. Thanks for the clarification. Better message sounds nice!

**Comment by teodorlu** [comment]
PS. Thanks for Insomnia! Really useful, just found out you made it!

**Comment by DavidAnson** [comment]
😀 Thanks very much!
